js/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte

```diff
@@ -78,39 +78,52 @@
 	class="flex flex-col w-full max-w-full
 	{isMaximized ? 'fixed inset-0 bg-white p-12 z-20' : ''}"
 >
-	{#if isMaximized}
-		<button class="absolute top-6 right-12" on:click={onClickMaximizeBtn}>
-			<IconCross classNames="text-xl text-gray-500 hover:text-black" />
-		</button>
-	{/if}
-	<WidgetHeader {noTitle} pipeline={model.pipeline_tag}>
-		{#if !!inputGroups.length}
-			<div class="ml-auto flex gap-x-1">
-				<!-- Show samples selector when there are more than one sample -->
-				{#if inputGroups.length > 1}
-					<WidgetInputSamplesGroup
-						bind:selectedInputGroup
+	{#if modelLoadInfo.state == "TooBig"}
+		<p class="text-sm text-gray-500">
+			Model is too large to load onto the free Inference API. To try the model,
+			launch it on <a
+				class="underline"
+				href="https://ui.endpoints.huggingface.co/mishig/new?repository={encodeURIComponent(
+					model.id
+				)}">Inference Endpoints</a
+			>
+			instead.
+		</p>
+	{:else}
```

Comment:
Don't display the widget until you know the modelLoadInfo, or it will (blink) ugly in the UI.

Reply:
handled in https://github.com/huggingface/hub-docs/pull/935/commits/61c6ec222f6bce053dd883f66ea6b3bf96f86264